RecoLocalCalo/Configuration/python/hgcalLocalReco_cff.py

```diff
@@ -7,8 +7,10 @@
 # (for now until global reco is going with some sort of clustering)
 from RecoParticleFlow.PFClusterProducer.particleFlowRecHitHGC_cff import *
 from RecoParticleFlow.PFClusterProducer.particleFlowClusterHGC_cfi import *
+from RecoLocalCalo.HGCalRecProducers.hgcalLayerClusters_cfi import *
```

Comment:
Could the `import *` be avoided please? Importing specific modules instead of "*" is strongly recommended.